tests/test_database_manager.py

```diff
@@ -0,0 +1,150 @@
+
+from . import setup_test
+import unittest
+import database_connector
+import database_manager
+import product_manager
+import helper
+import image_manager
+
+
+class TestDatabaseManager(unittest.TestCase):
+    @classmethod
+    def setUpClass(self):
+        self.database_connector = database_connector.DatabaseConnector()
+        self.item_manager = database_manager.ItemManager(
+            self.database_connector)
+        self.tag_manager = database_manager.TagManager(
+            self.database_connector)
+
+    def setUp(self):
+        setup_test.create_test_database_setup(self.database_connector)
+
+    def test_add_item(self):
+        # arrange
+        item_id = helper.get_id()
+
+        # act
+        self.item_manager.add_item(item_id, item_id, item_id)
+
+        # assert
+        self.assertTrue(setup_test.assertCount(
+            self.database_connector, 'Items', 1))
```

Comment:
Either move the `assertCount` into this class, and write the `assertTrue` in that function, or rename this function to `countRows` or something. (and use the assertEquals instead of assertTrue) Currently it is not asserting anything in the conventional sense, so the function might trick the coder into thinking something is actually asserted.

Reply:
updated function to count_rows